guide/05-working-with-the-spatially-enabled-dataframe/part2_data_io_reading_data.ipynb

```diff
@@ -0,0 +1,5229 @@
+{
```

Comment:
<p>The example below shows how the <code><a href="https://developers.arcgis.com/python/api-reference/arcgis.gis.toc.html?highlight=gis%20content%20get#arcgis.gis.ContentManager.get" target="_blank">get()</a></code> method can be used to retrieve an ArcGIS Online <code><a href="https://developers.arcgis.com/python/api-reference/arcgis.gis.toc.html?highlight=gis%20content%20get#item" target="_blank">item</a></code> and how the <code><a href="https://esri.github.io/arcgis-python-api/apidoc/html/arcgis.gis.toc.html#layer" target="_blank">layers</a></code> property of the <code>item</code>  can be used to access the data.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>